package.json

```diff
@@ -83,12 +83,12 @@
   "devDependencies": {
     "@babel/core": "^7.5.5",
     "@babel/preset-env": "^7.5.5",
-    "@bigcommerce/eslint-config": "^1.0.0",
-    "@bigcommerce/tslint-config": "^4.0.1",
+    "@bigcommerce/eslint-config": "^1.0.1",
```

Comment:
could we upgrade this to v2.6.1?

Reply:
Nice catch @DeveloperAlan. @bc-peng I'm happy for you to upgrade to the latest version in a follow-up PR if the diff becomes too big.

Reply:
Hi @DeveloperAlan , @DeveloperAlan . I think `v1.0.1` is the last `@bigcommerce/eslint-config` that can work with `typescript 3.9`. The `^2.0` requires `typescript @^4.0` as stated in its [package.json](https://github.com/bigcommerce/eslint-config/blob/a919ee7c6d4b04e4ad12604852423121dae68921/packages/eslint-config/package.json#L46). Also, below is a warning I'm receiving after running `npm install`.
> npm WARN @bigcommerce/eslint-plugin@1.1.1 requires a peer of typescript@^4.0.0 but none is installed. You must install peer dependencies yourself.

Reply:
FYI, we'll need to eventually upgrade the version of TS to at least 4.1 so it's compatible with the version of NX we've chosen. See https://github.com/bigcommerce/checkout-sdk-js/pull/1492

Reply:
Yes but I think its good to proceed with this PR, and create a ticket and upgrade TS + Lint in a follow up/another PR.